src/shared/efivars.c

```diff
@@ -152,9 +153,10 @@ static int get_os_indications(uint64_t *os_indication) {
                  * efi_reboot_to_firmware_supported() above. */
                 *os_indication = 0;
                 return 0;
-        } else if (r < 0)
+        }
```

Comment:
I don't see how that is an improvement. It is longer, and if anything it makes the code more brittle, because if the `return` is removed later on, this part needs to be updated too to undo this change. When the blocks are single-line, this doesn't matter so much, but with a longer block this change reduces legibility.

Reply:
force pushed a new version now where i dropped that... I am a fan of an "exit early and forget it" strategy when dealing with errors, and hence would always replace "else if" by "if" if the earlier branch all ended in a "return", simply so that as little "structural context" is carried on down to the rest of the function as possible, and we can forget everything we can forget about what came before...

But dunno, I don't care too much about this, hence dropped the hunk now, and force pushed again. PTAL.